template/{{.input_root_dir}}/.github/workflows/{{.input_project_name}}-bundle-cd-prod.yml.tmpl

```diff
@@ -17,6 +17,8 @@ defaults:
 env:
   {{ if (eq .input_cloud `aws`) -}}
   DATABRICKS_TOKEN: {{`${{ secrets.PROD_WORKSPACE_TOKEN }}`}}
+  {{- else if (eq .input_cloud `gcp`) -}}
```

Comment:
Can we instead combine with the above `if` condition? Or better yet, since we guarantee a strict enum, we can just do ```{{- if (eq .input_cloud `azure`) -}}``` \<azure-code\> `{{ else }}` \<shared-aws-gcp-code\>